rmf_dashboard_resources/office/main.json

```diff
@@ -1 +0,0 @@
-{"dispensers": {"coke_ingestor": {"icons": {"coke_ingestor": ""}, "location": {"x": 20.95, "y": -6.95, "yaw": 0, "level_name": "L1"}}, "coke_dispenser": {"icons": {"coke_dispenser": ""}, "location": {"x": 16.85, "y": -4.8, "yaw": 0, "level_name": "L1"}}}, "robots": {"deliveryRobot": {"icons": {"deliveryRobot": "/robots/deliveryRobot/deliveryRobot.png"}, "places": {}}, "tinyRobot": {"icons": {"tinyRobot": "/robots/tinyRobot/tinyRobot.png"}, "places": {"supplies": [], "tinyRobot2_charger": [], "coe": [], "tinyRobot1_charger": [], "hardware_2": ["coke_ingestor"], "cubicle_2": [], "pantry": ["coke_dispenser"], "station_1": [], "lounge": [], "cubicle_1": [], "hardware_1": [], "station_2": []}}}}
```

Comment:
Consequences of removing the main.json file. 
When the RomiDashboard gets the `rmf_dashboard_resources/office/` folder, the main.json file will not be present, so the dashboard without the docker-backend will not be able to load the resources as the file does not exist.

Possible solutions:
1. Also copy the python script `build_configuration.py` and run it on RomiDashboard to generate the main.json file. This would not be so great since we are adding a dependency on another language (python) to the project.
2. Create a node script that does the same as  `build_configuration.py` and we can store it and execute it directly in Romi-dashboard.

What do you think about this?

Reply:
>Consequences of removing the main.json file.
When the RomiDashboard gets the rmf_dashboard_resources/office/ folder, the main.json file will not be present, so the dashboard without the docker-backend will not be able to load the resources as the file does not exist.

good point, maybe we should commit `main.json` to support downloading from git.

Reply:
> > Consequences of removing the main.json file.
> > When the RomiDashboard gets the rmf_dashboard_resources/office/ folder, the main.json file will not be present, so the dashboard without the docker-backend will not be able to load the resources as the file does not exist.
> 
> good point, maybe we should commit `main.json` to support downloading from git.

+1